frontend/packages/integration-tests-cypress/views/list-page.ts

```diff
@@ -35,6 +35,19 @@ export const listPage = {
         cy.byLegacyTestID('dropdown-button').click();
       });
     },
+    clickFilterDropdown: () => {
+      cy.get('.pf-c-toolbar__content-section').within(() => {
+        cy.get('button.pf-c-select__toggle').click();
```

Comment:
Wherever possible we should add `data-test` attributes and select on those rather than using classnames or button labels. (Not just here, but in other selectors in the PR.)

Reply:
Unfortunately, these are inner components of the PatternFly filter/toolbar, so we don't have access to add IDs.

Reply:
Actually, there is already a legacy data id for the filter dropdown, you should be able to use:
```
clickFilterDropdown: () => {
      cy.byLegacyTestID('filter-dropdown-toggle').within(() => {
        cy.get('button').click();
```

Reply:
ok, let me update

Reply:
updated